Public/Src/Utilities/Utilities/VmCommandProxy/VmInitializer.cs

```diff
@@ -25,7 +26,7 @@ public class VmInitializer
         public string VmCommandProxy { get; }
 
         private readonly string m_userName;

```

Comment:
>m_userName; [](start = 32, length = 11)

Remove userName and password from this class, and obtain them using the lowprivilege account utils.